shell/platform/tizen/tizen_window_ecore_wl2.h

```diff
@@ -34,6 +34,8 @@ class TizenWindowEcoreWl2 : public TizenWindow {
 
   void* GetRenderTarget() override { return ecore_wl2_egl_window_; }
 
+  void* GetNativeHandle() override { return ecore_wl2_window_; }
+
   void* GetRenderTargetDisplay() override { return wl2_display_; }
```

Comment:
Please change the order. `GetRenderTargetDisplay` should be defined as close as possible to `GetRenderTarget`.
```suggestion
  void* GetRenderTargetDisplay() override { return wl2_display_; }

  void* GetNativeHandle() override { return ecore_wl2_window_; }
```

Reply:
Applied the modifications that suggested.
tizen_window_element.h also has the same content, so modified it.
Thank you for your proposal.